Source/DataSources/GroundGeometryUpdater.js

```diff
@@ -51,9 +64,25 @@ define([
             get: function() {
                 return this._zIndex;
             }
+        },
+
+        terrainOffsetProperty: {
+            get: function() {
+                return this._terrainOffsetProperty;
+            }
         }
     });
 
+    /**
+     * @param {Entity} entity
```

Comment:
Since none of the other private functions in this file have doc, I would just remove it for now for consistency. (if you really want to keep it, we should flesh it out).

Reply:
Looks like the doc that's here is wrong anyway.